packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -957,6 +953,15 @@ mixin SchedulerBinding on BindingBase {
     return _lastRawTimeStamp;
   }
 
+  /// The current frame number.
+  ///
+  /// In release mode, this always return `-1`.
+  int get debugFrameNumber {
+    if (!kReleaseMode) {
+      return _debugFrameNumber;
```

Comment:
since _debugFrameNumber is only incremented in debug mode, should we return -1 for profile mode too?